testapp/settings.py

```diff
@@ -91,6 +92,16 @@
 # LOG OUTGOING REQUESTS
 #
 LOG_OUTGOING_REQUESTS_DB_SAVE = True
+LOG_OUTGOING_REQUESTS_DB_SAVE_BODY = True
+LOG_OUTGOING_REQUESTS_CONTENT_TYPES = [
+    "text/*",
+    "application/json",
+    "application/xml",
+    "application/soap+xml",
+]
+LOG_OUTGOING_REQUESTS_EMIT_BODY = True
+LOG_OUTGOING_REQUESTS_MAX_CONTENT_LENGTH = 524_288  # 0.5 MB
+LOG_OUTGOING_REQUESTS_LOG_BODY_TO_STDOUT = True
```

Comment:
I thought this setting should be replaced with `LOG_OUTGOING_REQUESTS_EMIT_BODY`?